assessments/0002-notary-project.md

```diff
@@ -0,0 +1,196 @@
+   - This is an area for improvement on both V1 and V2 docs. Potentially consider reorganizing the whole docset with users in mind: dividing the content in a concepts, tasks, and tutorials model. 
+
+#### New user content
+
+- Getting Started Guide is clearly labelled: https://github.com/theupdateframework/notary#getting-started-with-the-notary-cli though notably only in v1 of their docs, not v2. How does one get started with v2?
+- The v2 overview has getting started/CLI steps for an entirely different set of functions: https://github.com/notaryproject/notaryproject#notary-v2-overview. Are these in addition to or in replacement of the v1 getting started steps?
+- Where do users go after the getting started steps? Is there anything else about the tool they need to understand?
+- Sample code looks great!
+
+#### Content maintainability
+
+- Regarding searchability, as the documentation is all in GitHub currently, it is searchable; however, as a part of setting up a website, we recommend using a tool like Google, Algolia, or Lunr to provide search functionality. 
+  
+- Is the Notary Project planning on localization/internationalization? If so, we recommend starting the site with a directory structure which supports localization (Hugo supports this very well out of the box).
+
+- Regarding versioning, does V1 lead directly to V2?
+   - If not, then V1 and V2 should be brought in as separate document sets, and V2 should be set up with a versioning process (allowing V1 to exist as a sort of archive).
+   - If so, then V1 can act as a base for the site and no separation of versions need be applied.
+
+#### Content creation processes
+
+<!-- borrowed from @celestehorgan's excellent work in 0001-contour.md -->
+- The following are ideas to consider going forward with a docs repo. As a content creator:
+   - If I write documentation, who verifies that it's technically accurate? 
+   - If I'm not a native english speaker, who can I ask for help with grammar and language review? 
+   - If I'm a trained technical writer and want to rearrange, create new, or split existing topics, who do I tag in issues/on Slack for comment?
+   - As a developer user, how can I ensure that content is accurate and up to date? Are alpha and beta features flagged? What happens when content (+ functionality) gets deprecated or changed in a major way? 
+   - Who reviews documentation PRs? 
+
+### Recommendations
+
+#### Information Architecture
+
+The assumption is that V2 is the primary project moving forward, and that a Minimum Viable Product (MVP) for a website/documentation will focus on Notary V2 users.
+- Bring the nv2 cli prototype documentation into the main documentation when conosolidating the content on the website.
+- V1 Docs will need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
```

Comment:
Asking genuinely – what does IA mean?

Reply:
"Information Architecture" or, basically: "what order is the content in, where are the topics/subtopics, and is that ordering/grouping good?"

Reply:
Thanks, Celeste! @nate-double-u what do you think of specifying what acronyms stand for the first time we use them? (I frequently forget this myself.)

Reply:
> Thanks, Celeste! @nate-double-u what do you think of specifying what acronyms stand for the first time we use them? (I frequently forget this myself.)

Good suggestion, I'll do that